README.md

```diff
@@ -147,7 +156,42 @@ OpenShift server):
 $ oc login [URL]
 ```
 
-Now, you can create a fresh OpenShift project:
+### Installing CI System in OpenShift using Ansible
+
+NOTE: Ansible will be running all of the commands on `localhost`, instead of
+executing the tasks remotely in the cluster.  That is, the Ansible playbook
+will be using the `oc` OpenShift command line tool, and the python OpenShift
+API library, to execute API calls in the OpenShift cluster to provision and
+configure the resources in the OpenShift cluster.
```

Comment:
This sounds good. Maybe just move the login command here and to the other section for more visibility.